### ISSUE TITLE
dangerous_statement, dynamic sql

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#2541]
dont report for hardcoded dynamic values,

```abap
        SELECT * FROM ssyntaxstructure                    "#EC CI_SUBRC
          INTO TABLE gt_syntax WHERE (`proglang = 'A'`)
                                  OR (`proglang = ''`)
                                  OR (`proglang IS NULL`). "proglang = 'B' is BDL, not ABAP
```

![image](https://user-images.githubusercontent.com/5888506/168996542-8718d560-a0d8-4246-a528-2c8f75055cab.png)
